Neuro/Movement/MovementHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Neuro.DependencyInjection;
+using Neuro.Tasks;
+using UnityEngine;
+
+namespace Neuro.Movement;
+
+public class MovementHandler : IMovementHandler
+{
+    public IContextProvider Context { get; set; }
+
+    public Vector2 LastMoveDirection { get; private set; }
+    public Vector2 DirectionToNearestTask { get; private set; }
+
+    public LineRenderer Arrow { get; set; }
+
+    public Vector2? GetForcedMoveDirection(Vector2 actualDirection)
```

Comment:
I've been curious for a while now, why did you chose to use a `Vector2?` instead of keeping to the `bool` return + `ref/out` param like in main?